exonum/src/explorer/mod.rs

```diff
@@ -53,58 +53,69 @@ pub struct TxInfo {
     pub location: TxLocation,
     /// Proof that transaction really exist in database
     pub proof_to_block_merkle_root: ListProof<Hash>,
```

Comment:
Not related to the PR, but perhaps just `proof` would be a better name?

Reply:
It could be confusing given that we might want to add a proof for the transaction execution status. I'd suggest `location_proof` for this proof and `status_proof` for the corresponding proof (if/when it appears).